drivers/f90/driver.f90

```diff
@@ -96,7 +96,9 @@ PROGRAM DRIVER
          IF (cmdbuffer == "-u") THEN ! flag for unix socket
             inet = 0
             ccmd = 0
-         ELSEIF (cmdbuffer == "-h") THEN ! read the hostname
+         ELSEIF (cmdbuffer == "-h") THEN ! read the hostname (deprecated)
+            ccmd = 1
+         ELSEIF (cmdbuffer == "-a") THEN ! read the hostname (address)
```

Comment:
...this also seems unrelated to the current PR and probably should be discussed separately (because it's changing the CLI of the driver!)

Reply:
it's related - argparse has special meaning for -h. This doesn't change (yet) the CLI of the fortran driver, also adds an "alias" so that -a works the same for both